ta/pkcs11/src/processing.c

```diff
@@ -1003,3 +1003,181 @@ enum pkcs11_rc entry_release_active_processing(struct pkcs11_client *client,
 
 	return PKCS11_CKR_OK;
 }
+
+enum pkcs11_rc entry_wrap_key(struct pkcs11_client *client,
+			      uint32_t ptypes, TEE_Param *params)
+{
+	const uint32_t exp_pt = TEE_PARAM_TYPES(TEE_PARAM_TYPE_MEMREF_INOUT,
+						TEE_PARAM_TYPE_NONE,
+						TEE_PARAM_TYPE_MEMREF_OUTPUT,
+						TEE_PARAM_TYPE_NONE);
+	TEE_Param *ctrl = params;
+	enum pkcs11_rc rc = PKCS11_CKR_GENERAL_ERROR;
+	struct serialargs ctrlargs = { };
+	struct pkcs11_session *session = NULL;
+	struct pkcs11_attribute_head *proc_params = NULL;
+	struct pkcs11_object *wrapping_key = NULL;
+	struct pkcs11_object *key = NULL;
+	struct obj_attrs *req_attrs = NULL;
+	uint32_t wrapping_key_handle = 0;
+	uint32_t key_handle = 0;
```

Comment:
`CK_INVALID_HANDLE` is for the purpose of telling invalid handle.

Reply:
0 is the invalid handle identifier as per _ta/pkcs11/src/handle.c_ implementation.
I'm fine with 0 as is.
We can create a specific P-R to `#define HANDLE_INVALID 0` in _handle.h_ and use it in  ta/pkcs11 source files.

Reply:
Leaving as it is for now.